core/handlers/slave/vote.py

```diff
@@ -16,6 +16,15 @@ def __is_user_voted(db, user_id, original_chat_id, message_id):
 
     return False
 
+@coroutine
+def __prev_vote(db, user_id, original_chat_id, message_id):
```

Comment:
It would be better to combine `__is_user_voted()` and `__prev_vote()`, IMO. Because this functions, basically, are the same.

Reply:
thought so too, I just didnt want to touch previously written logic, thats all.
Will change